src/System.Net.WebSockets/src/System/Net/WebSockets/WebSocketException.cs

```diff
@@ -122,11 +122,6 @@ public WebSocketException(string message, Exception innerException)
         {
         }
 
-        public override void GetObjectData(SerializationInfo info, StreamingContext context)
```

Comment:
@danmosemsft What are the other classes in .NET Core doing regarding removing (or not removing) the GetObjectData() overrides.  This might be breaking to remove them.  So, I can simply leave the code as-is (which is a no-op since it just calls the base.GetObjectData).